docs/reference/cluster/enroll-node.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[cluster-enroll-node]]
+=== Enroll Node API
+++++
+<titleabbrev>Enroll Node</titleabbrev>
+++++
+
+Allows a new node to join an existing cluster with security enabled.
+
+[[cluster-enroll-node-api-request]]
+==== {api-request-title}
+
+`GET /_cluster/enroll_node`
```

Comment:
Isn't it POST?

Reply:
Nope, it is a GET. It doesn't change anything in the node, it just retrieves information. It might even make sense to rename this to Get Enrollment Information or something similar